layouts/guides/list.html

```diff
@@ -7,24 +7,7 @@ <h1>GSA Tech Guides</h1>
     {{ .Content }}
 
     <div class="grid-row grid-gap">
-
-      <div class="tablet:grid-col-3">
-        <div class="position-sticky top-3 margin-top-3">
-          <form class="usa-form">
-            <fieldset class="usa-fieldset guides-filter">
-              <legend class="usa-legend">Filter by Category</legend>
-              {{ range .Pages.GroupByParam "category" }}
-                <div class="usa-checkbox">
-                  <input class="usa-checkbox__input guides-filter-category" id="category-{{ .Key | urlize }}" type="checkbox" name="categories" value="{{ .Key | urlize }}" checked>
-                  <label class="usa-checkbox__label" for="category-{{ .Key | urlize }}">{{ .Key }}</label>
-                </div>
-              {{ end }}
-            </fieldset>
-            <button type="submit" class="usa-sr-only">Filter</button>
-          </form>
-        </div>
-      </div>
-
+      
       <div class="tablet:grid-col-9">
         {{ range .Pages.GroupByParam "category" }}
```

Comment:
I think it's fine to keep this even thought there is only one category…but just pointing out that if we'll never have anything other than Agile, we could simplify this too.